core/tauri-utils/src/config.rs

```diff
@@ -1049,9 +1071,12 @@ pub struct BundleConfig {
   /// Whether Tauri should bundle your application or just output the executable.
   #[serde(default)]
   pub active: bool,
-  /// The bundle targets, currently supports ["deb", "rpm", "appimage", "nsis", "msi", "app", "dmg", "updater"] or "all".
+  /// The bundle targets, currently supports ["deb", "rpm", "appimage", "nsis", "msi", "app", "dmg"] or "all".
   #[serde(default)]
   pub targets: BundleTarget,
+  #[serde(default)]
+  /// Produce updaters and their signatures or not
+  pub updater: Updater,
```

Comment:
After some thinking I think we should call this `createUpdaterArtifacts`. This makes it clear that it generates files to be used by the updater and won't confuse users as to why this option exists here and not inside `plugins > updater`

Reply:
cc @FabianLars @lucasfernog